CMake/FindITKUtil.cmake

```diff
@@ -55,7 +55,7 @@ macro(FindITKUtil ITK_VAR_PREFIX )
     ${ARGN}
     )
   list(REMOVE_DUPLICATES ITK_INCLUSION_MODULES)
-  find_package(ITK 4.5 COMPONENTS ${ITK_INCLUSION_MODULES} REQUIRED)
+  find_package(ITK COMPONENTS ${ITK_INCLUSION_MODULES} REQUIRED)
```

Comment:
This indicates the minimum version.  This prevents finding ITK version 4.4 and earlier that would not work.  I'll be happy to merge once this is reverted back.